lib/git_pair/cli.ex

```diff
@@ -24,10 +29,19 @@ defmodule GitPair.CLI do
     |> print_result
   end
 
-  defp parse_args(args), do: OptionParser.parse(args, switches: @switches, aliases: @aliases)
+  defp parse_args(args), do: OptionParser.parse(args, strict: @switches, aliases: @aliases)
 
   defp parse_command({_, [action | args], _}), do: {action, args}
 
+  defp execute_command({"help", []}) do
+    @help
+    |> Enum.map(fn detail ->
+      {command, explanation} = detail
+      "#{command}: #{explanation}"
+    end)
+    |> (&{:ok, Enum.join(&1, "\n")}).()
```

Comment:
What do you mean when doing this expression `(...).()`?

Reply:
@leandroh this means that the parameter of previous `pipe` will be passed to this self executed function:

The `&` is used to create an anonymous function and the wrapping `(&).()` to execute it, using the first argument received by previous pipe (`&1`), which is the list of commands with explanation (`["add: Adds collaborators", "etc: Lorem ipsum"]`).